dotnet/src/Connectors/Connectors.Memory.Weaviate/WeaviateQueryBuilder.cs

```diff
@@ -81,7 +81,8 @@ public static string BuildQuery<TRecord>(
     {
         var vectorsQuery = GetVectorsPropertyQuery(queryOptions.IncludeVectors, hasNamedVectors, model);
 
-        var sortPaths = string.Join(",", queryOptions.OrderBy.Values.Select(sortInfo =>
+        var orderBy = queryOptions.OrderBy?.Invoke(new()).Values;
+        var sortPaths = orderBy is not { Count: > 0 } ? "" : string.Join(",", orderBy.Select(sortInfo =>
```

Comment:
nit: invert negation for slightly better clarity

```suggestion
        var sortPaths = orderBy is { Count: 0 } ? "" : string.Join(",", orderBy.Select(sortInfo =>
```

Reply:
Will `orderBy is { Count: 0 }` handle the case where `orderBy` is `null`? (that is the only reason I decided to use negation here)

Reply:
Ah I see - no, but then you can do `order by is null or { Count: 0 }` for that (I just hate double negation)